runtimes/tensorflow/ubi9-python-3.12/Dockerfile.cuda

```diff
@@ -121,6 +121,11 @@ RUN yum install -y \
 # Set this flag so that libraries can find the location of CUDA
 ENV XLA_FLAGS=--xla_gpu_cuda_data_dir=/usr/local/cuda
 
+# hdf5 is needed for h5py
+RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
+    dnf install -y hdf5-devel && \
+    dnf clean all
```

Comment:
@coderabbitai could it be that because we're locking for x86-64 (on amd64 machine), we are actually ignoring an existing arm64 wheel for h5py that would obliviate the need for a -devel package here?

Reply:
@coderabbitai create github ticket to look into this later, we have to merge, merge, merge now now!